unit_tests.py

```diff
@@ -45,5 +80,15 @@ def test_joke_of_the_day(self):
                 assert False
             jokes_ids.append(joke.id)
 
+    def test_sorting_method(self):
+        """test sorting method in models"""
+        testing_joke = Joke.query.filter_by(id=1).first()
+        insert_joke_reactions(testing_joke)
+        testing_order = testing_joke.order_reactions()
+        result_order = [("funny", 3), ("unamused", 2), ("smile", 1)]
+        assert testing_order == result_order
```

Comment:
Do you need a variable for this? You could simply use the list literal in the `assert`